docs/3. Product Features/01. Chats.md

```
@@ -62,6 +62,14 @@ A green phone-like icon towards the right bottom end is present which acts as a
 
 <img width="281" alt="Screenshot 2024-02-29 at 12 13 21 PM" src="https://github.com/glific/docs/assets/143380171/0bbf03af-1c81-4af3-a834-b0719d7b7d93"/>
 
+**Check if contact is not opted in**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)